src/ts_src/index.js

```diff
@@ -0,0 +1,169 @@
+"use strict";
```

Comment:
`src/ts_src` itself seems unnecessary? (Maybe you accidentally built it weird once and committed?)

This index.js is similar to `src/index.js`

Reply:
https://github.com/bitcoinjs/bip39/pull/104/commits/d84e2821f35582bbb718b1e638198d35d34b8f14